plutus-book/doc/multi/multi.adoc

```diff
@@ -1,5 +1,5 @@
 [#multi]
-= Multi-stage contracts
+= Multi-Stage Contracts
```

Comment:
We're using sentence case for titles everywhere else, I think we should stick with it.

Reply:
We have "UTxO Style Accounting", Welcome to Plutus - A New Language", "Extended UTxO Model"...